src/tests/examples/output_compare_test.py

```diff
@@ -0,0 +1,659 @@
+    return output_values
+
+def remove_outputs(study_path):
+    output_path = study_path / 'output'
+    files = glob.glob(str(output_path))
+    for f in files:
+        shutil.rmtree(f)
+
+def launch_solver(solver_path, study_path, use_ortools = False, ortools_solver = "sirius"):
+    # Clean study output
+    remove_outputs(study_path)
+    
+    solver_path_full = str(Path(solver_path).resolve())
+
+    command = [solver_path_full, "-i", str(study_path)]
+    if use_ortools:
+        command.append('--use-ortools')
+        command.append('--ortools-solver='+ortools_solver)
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=None)
+    output = process.communicate()
+
+    # TODO check return value
+    assert "Solver returned error" not in output[0].decode('utf-8')
+
+def generate_reference_values(solver_path, path, use_ortools, ortools_solver):
+
+    enable_study_output(path,True)
+
+    reference_path = path / 'reference'
+    os.makedirs(reference_path, exist_ok=True)
+    launch_solver(solver_path,path, use_ortools, ortools_solver)
+
+    output_path = path / 'output'
+
+    result_dir = find_output_result_dir(output_path)
+    shutil.copytree(result_dir, reference_path / 'output' / result_dir.name)
+
+def run_study(solver_path, path, use_ortools, ortools_solver):
+    # Launch antares-solver
+    launch_solver(solver_path, path, use_ortools, ortools_solver)
+    
+def enable_study_output(study_path, enable):
+    st = study(str(study_path))
+    st.check_files_existence()
+
+    synthesis_value = "true" if enable else "false"
+    st.set_variable(variable = "synthesis", value = synthesis_value, file_nick_name="general")
+
+def compare_directory(result_dir, reference_dir):
+    assert (result_dir.is_dir())
+    assert (reference_dir.is_dir())
+    dir_list = []
+    for x in result_dir.iterdir():
+        if x.is_dir():
+            if x.name != 'grid':
+                compare_directory(x, reference_dir / x.name)
+        else:
+
+            if x.name != 'id-daily.txt' :
+                reference_values = get_output_values(reference_dir / x.name)
+                output_values = get_output_values(x)
+
+                np.testing.assert_allclose(reference_values, output_values, rtol=1e-4, atol=0,equal_nan=True, err_msg="values dismatch in " + str(reference_dir / x.name), verbose=True)
+
+def check_output_values(path):
+    result_dir = find_output_result_dir(path / 'output')
+    reference_dir = find_output_result_dir(path / 'reference')
+    compare_directory(result_dir, reference_dir)
+
+## TESTS ##
+@pytest.mark.short
+def test_001_one_node_passive(use_ortools, ortools_solver, solver_path):
+    study_path = ALL_STUDIES_PATH / "short-tests" / "001 One node - passive"
+    enable_study_output(study_path, True)
+    run_study(solver_path, study_path, use_ortools, ortools_solver)
+    enable_study_output(study_path, False)
```

Comment:
I'm not sure this line is useful : the solver has already run, config files modifications won't be taken into account.

Reply:
Studies in https://github.com/AntaresSimulatorTeam/Antares_Simulator_Tests don't have output enabled.
We must use  `enable_study_output(study_path, True) `before run and disable after